### PR TITLE
DOCS-10811 May release

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
DOCS-10811 May release

### CODE REVIEW

**Comment by wwarner-salesforce** [comment]
Questions:
- [x] When using guided setup, is it necessary to configure the guest user and member user permissions?
- [x] The upgrade instructions say to make the permissions changes "If you are upgrading from a package version before 1.80.5". To confirm, this means users upgrading from 1.80.5 to 1.82 should not make any permissions changes? Or users upgrading from 1.80.5. to 1.82 should make the permissions changes, and the text should be "If you are upgrading from a package version before 1.82"?

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 1]
```diff
@@ -161,7 +161,8 @@ You can specify values for the following parameters of the API Console:
 API:: Specifies the API to display.
 Show Documentation Pages:: Shows the API's documentation pages.
 Expand endpoints on start:: Expands the documentation of the API endpoints when the page loads.
-Enable resource filtering:: Shows a search bar in the left navigation so users can filter the API resources and show only the resources that match their search terms.
+Enable resource filtering:: Shows a search bar on the navigation panel so users can filter the API resources and show only the resources that match their search terms.
+Show endpoints toggle button:: Shows a button on the navigation panel to expand or collapse all endpoints.
 Remove overview items from navigation:: Removes the overview links and shows the overview when the user clicks the endpoint name.
 Show menu action bar:: Shows the menu action bar with the *Quick Find* box and the button to expand or collapse all endpoints.
```
Comment (ferferrarazzo) [inline modules/ROOT/pages/acm-lightning-components.adoc]:
```suggestion this is no longer a option

```

File: modules/ROOT/pages/update-acm.adoc [hunk 2]
```diff
@@ -255,6 +256,41 @@ If you are upgrading from a package version before 1.80.1, update the permission
 .. Save.
 . Verify that your permissions match those in the xref:create-community.adoc#configure-profiles-and-permissions[Configure Profiles and Permissions] documentation.
 
+Return to the main list of upgrade instructions to continue.
+
+[[post-1805]]
```
Comment (ferferrarazzo) [inline modules/ROOT/pages/update-acm.adoc]: The upgrade instructions say to make the permissions changes "If you are upgrading from a package version before 1.80.5". To confirm, this means users upgrading from 1.80.5 to 1.82 should not make any permissions changes? Or users upgrading from 1.80.5. to 1.82 should make the permissions changes, and the text should be "If you are upgrading from a package version before 1.82"?

YES

File: modules/ROOT/pages/update-acm.adoc [hunk 3]
```diff
@@ -255,6 +256,41 @@ If you are upgrading from a package version before 1.80.1, update the permission
...
+... Click *Settings* > *General*.
+... Open the *Guest* profile.
+... Click *Object settings* > *AnypointAssetCustomFields*.
+... Ensure that *Read* is enabled and, in *Field Permissions*, ensure that all fields have *Read Access* enabled.
+... Click *Object settings* > *AnypointCustomFields*.
+... Ensure that *Read* is enabled and, in *Field Permissions*, ensure that all fields have *Read Access* enabled.
+... Click *Object settings* > *AnypointApiInstances*.
+... Ensure that the *AssetVersionId* field has *Read* access.
+... Save.
+. Verify that your permissions match those in the xref:create-community.adoc#configure-profiles-and-permissions[Configure Profiles and Permissions] documentation.
```
Comment (ferferrarazzo) [inline modules/ROOT/pages/update-acm.adoc:292]: We need to add this 2 new objects to the table
AnypointAssetCustomFields
AnypointCustomFields

**Review by ferferrarazzo — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Done. Thank you!

File: modules/ROOT/pages/update-acm.adoc (hunk 2 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/update-acm.adoc]: Done. Thank you!

File: modules/ROOT/pages/update-acm.adoc (hunk 3 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/update-acm.adoc:292]: Done. Thank you!

**Review by ferferrarazzo — APPROVED** [review]

**Pull request merged**